compiler/tests/generated/complex_types_concatenated.java

```diff
@@ -90,6 +90,21 @@ public Foo() {
         ((StructBondTypeImpl)BOND_TYPE).initializeStructFields(this);
     };
 
+    @Override
+    public boolean equals(Object o) {
+        if (!(o instanceof Foo)) return false;
+        
+        final Foo other = (Foo) o;
```

Comment:
I don't get one. `@generated` quiets the IDE on this; I don't think the default invocation of `javac` minds either way.